### ISSUE TITLE
Can't receive private messages inside a room

### ISSUE THREAD

**Issue by neurolit** [boothj5/profanity#598]
I'm using Profanity 0.4.6, and I've got a problem with private messages in a chat room : Profanity doesn't display the answers I receive. However I can see them in the XML Console.

Sending such messages works. Receiving does not.

Private messages work outside a chat room (directly from my roster).

Here is an example of what I see in the XML Console:

```
13:52:55 - RECV:
13:52:55 - <message id="purpled991a858" type="chat" to="me@jabber.fr/Home" 
           from="room@conference.jabber.fr/MyPal"><composing 
           xmlns="http://jabber.org/protocol/chatstates"/><x 
           xmlns="http://jabber.org/protocol/muc#user"/></message>
13:52:55 - 
13:52:56 - RECV:
13:52:56 - <message id="purpled991a859" type="chat" to="me@jabber.fr/Home" 
           from="room@conference.jabber.fr/MyPal"><active 
           xmlns="http://jabber.org/protocol/chatstates"/><body>I'm sending a test message</body><x 
           xmlns="http://jabber.org/protocol/muc#user"/></message>
13:52:56 - 
13:52:56 - RECV:
13:52:56 - <message id="purpled991a85a" type="chat" to="me@jabber.fr/Home" 
           from="room@conference.jabber.fr/MyPal"><active 
           xmlns="http://jabber.org/protocol/chatstates"/><x 
           xmlns="http://jabber.org/protocol/muc#user"/></message>
```

**Comment by boothj5** [comment]
The issue seems to be that the other occupant's client is attaching an element with the `http://jabber.org/protocol/muc#user` namespace to private messages.

There is no mention of this namespace for private messaging at http://www.xmpp.org/extensions/xep-0045.html#privatemessage, as it is used for sending presence or other information about a user (banning, kicking or sending invites for example).

However, I've added a fix so profanity should still display the message if other clients send the namespace.

I have tested locally, but if you are able to try out the latest code, let me know if the problem is resolved.
